frontend/src/Settings/ImportLists/ImportLists/AddImportListModalContent.js

```diff
@@ -0,0 +1,102 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { kinds } from 'Helpers/Props';
+import Alert from 'Components/Alert';
+import Button from 'Components/Link/Button';
+import FieldSet from 'Components/FieldSet';
+import LoadingIndicator from 'Components/Loading/LoadingIndicator';
+import ModalContent from 'Components/Modal/ModalContent';
+import ModalHeader from 'Components/Modal/ModalHeader';
+import ModalBody from 'Components/Modal/ModalBody';
+import ModalFooter from 'Components/Modal/ModalFooter';
+import AddImportListItem from './AddImportListItem';
+import styles from './AddImportListModalContent.css';
+import titleCase from 'Utilities/String/titleCase';
+
+class AddImportListModalContent extends Component {
+
+  //
+  // Render
+
+  render() {
+    const {
+      isSchemaFetching,
+      isSchemaPopulated,
+      schemaError,
+      listGroups,
+      onImportListSelect,
+      onModalClose
+    } = this.props;
+
+    return (
+      <ModalContent onModalClose={onModalClose}>
+        <ModalHeader>
+          Add List
+        </ModalHeader>
+
+        <ModalBody>
+          {
+            isSchemaFetching &&
+              <LoadingIndicator />
+          }
+
+          {
+            !isSchemaFetching && !!schemaError &&
+              <div>Unable to add a new list, please try again.</div>
+          }
+
+          {
+            isSchemaPopulated && !schemaError &&
+              <div>
+
+                <Alert kind={kinds.INFO}>
+                  <div>Sonarr supports multiple lists for importing Series into the database.</div>
+                  <div>For more information on the individual lists, click on the info buttons.</div>
```

Comment:
Looks like these info buttons don't have wiki pages yet, is this something we should copy from Radarr?

Reply:
Yea they will need to be created, we can use Radarr or Lidarr as we base for the page